app/services/candidates/registrations/behaviours/education.rb

```diff
@@ -14,9 +14,12 @@ module Education
           validates :degree_stage, inclusion: { in: :available_degree_stages }, if: -> { degree_stage.present? }
           validates :degree_stage_explaination, presence: true, if: :degree_stage_explaination_required?
           validates :degree_subject, presence: true
-          validates :degree_subject, inclusion: { in: :available_degree_subjects }, if: -> { degree_subject.present? }
-          validates :degree_subject, inclusion: [NO_DEGREE_SUBJECT], if: -> { degree_stage.present? && !degree_stage_requires_subject? }
-          validates :degree_subject, exclusion: [NO_DEGREE_SUBJECT], if: -> { degree_stage.present? && degree_stage_requires_subject? }
+          validates :degree_subject, inclusion: { in: :available_degree_subjects },
+            if: -> { degree_subject.present? && degree_stage_requires_subject? }
+          validates :degree_subject, inclusion: [NO_DEGREE_SUBJECT],
+            if: -> { degree_subject.present? && degree_stage.present? && !degree_stage_requires_subject? }
```

Comment:
This is a touch complex and repetitive, perhaps would make more sense in a separate method?

Reply:
complex yes, but repetative - no, each is a separate combination

I'll move each conditional to their own method then the name can reflect the behaviour

Reply:
Ah, I just meant combining `degree_subject.present? && degree_stage.present?` as that combination is used more than once

Reply:
Ah - see what you mean - I think there's probably more value in describing what the condition is supposed to be achieving so I've moved the second two into internal methods with names which hopefully explain this a little better